test/known_issues/test-vm-global-non-writable-properties.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+// https://github.com/nodejs/node/issues/10223
+
+require('../common');
+const assert = require('assert');
+const vm = require('vm');
+
+const ctx = vm.createContext();
+vm.runInContext('Object.defineProperty(this, "x", { value: 42 })', ctx);
+assert.strictEqual(ctx.x, undefined);  // Not copied out by cloneProperty().
```

Comment:
@bnoordhuis Right, I agree. Would you prefer it if I updated the test with `strictEqual(ctx.x, 42)` here?